src/navigation/slam_gridmap/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>slam_gridmap</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="claudiakiss005@gmail.com">yurikun1033</maintainer>
+  <license>TODO: License declaration</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+
```

Comment:
Probably can ditch the tests and the test folder (they're created automatically by ros)